src/libcrun/linux.c

```diff
@@ -4816,7 +4818,6 @@ libcrun_run_linux_container (libcrun_container_t *container, container_entrypoin
           else
             {
               init_status.idx_timens_to_join_immediately = i;
-              init_status.namespaces_to_unshare &= ~CLONE_NEWTIME;
```

Comment:
I prefer to keep it so it is clear that the namespace doesn't need to be created

Reply:
ok